politeiad/cmd/politeia/politeia.go

```diff
@@ -66,6 +66,8 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "\n actions:\n")
 	fmt.Fprintf(os.Stderr, "  identity          - Retrieve server "+
 		"identity\n")
+	fmt.Fprintf(os.Stderr, "  verify            - Verify a record "+
```

Comment:
Reason I chose to pass in the merkle root instead of manually passing the files+metadata to calculate it is that this verification is already done when creating a new proposal. So when retrieving a record from `d` with a `politeia` cmd, the merkle root returned is known to be correct.

Manually passing down the files+metadata let's us verify the merkle root as well, but imo it's a bit redundant to verify this again, and would add a lot of verbosity to the command.

Reply:
That makes sense and is indeed a much better UX.